CryptoPrices/Sources/Data/Sources/Repositories/CoinRepository.swift

```diff
@@ -8,6 +8,7 @@
 import Entities
 import NetworkCore
 import RepositoryProtocol
+import NetworkExtension
```

Comment:
The `Repository` can work with any type that conform to `CoinAPIProtocol` (hence the need of `import NetworkCore`) It doesn't depend on a specific implementation. So we should omit the `import NetworkExtension` here.

Reply:
nice catch! I was testing different approaches on how to manage the shared enum with the current modular architecture but I forgot to clean up this one, will remove 👍